.github/workflows/rust.yml

```diff
@@ -0,0 +1,53 @@
+name: Rust-DataFusion
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    container:
+      image: amd64/rust
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Rust Toolchain
+        uses: actions-rs/toolchain@v1
+        id: rust-toolchain
+        with:
+          toolchain: stable
+          override: true
+          components: rustfmt, clippy
+      - name: Run
+        run: cargo fmt --manifest-path ./dask_planner/Cargo.toml --all -- --check
```

Comment:
IIUC we can remove this because it's being handled as part of the pre-commit checks now, correct?